app/selectors.js

```diff
@@ -249,6 +250,49 @@ export const transactions = createSelector(
 export const homeHistoryTransactions = createSelector(
   [transactionsNormalizer, get(["grpc", "recentTransactions"])], apply
 );
+
+export const spendableAndLockedByDay = createSelector(
+  [transactions],
+  (transactions) => {
+    let valuesByDate = {};
+    let spendableTotal = 0;
+    let lockedTotal = 0;
+    for(let i=transactions.length-1; i>=0; i--) {
+      let transaction = transactions[i];
+      const time = tsToYYYYMMDD(transaction.txTimestamp);
```

Comment:
Instead of using a fixed format, for displaying purposes I'd use a function to convert the timestamp to `Date.toLocaleDateString()` as that is the format for the user's OS.

Eventually we could move to use moment.js as it has better formatting capabilities, but that's not needed right now.